packages/shared/src/components/checklist/actionUtils.ts

```diff
@@ -12,10 +12,3 @@ export const getEditActions = (squad: Squad): Action[] => {
     ? [{ type: ActionType.EditSquad, completedAt: new Date() }]
     : [];
 };
-
-export const getPublicActions = (squad: Squad): Action[] => {
```

Comment:
this one is already removed in this branch?
https://github.com/dailydotdev/apps/blob/8ffbe0687a507702747ad9e26efeec642a7e33a9/packages/shared/src/hooks/useSquadChecklist.tsx#L146

Reply:
Right. Been looking for this. Thank you! 🚀

Reply:
Ah, it has been deleted from this file already. Thought the link was towards this PR. We should be good.